.github/workflows/build-backend.yaml

```
@@ -0,0 +1,44 @@
+name: "Backend build check"
+
+on:
+  pull_request:
+    paths:
+      - '**.go'
+  push:
+    branches:
+      - 'main'
```

single quotationいらない？
```suggestion
      - main
```

---

全体的にyaml内のquotationの使い方がHirai-sanと違うからどっちかに統一する？

---

基本的にいらんはずやけど、[GitHubのテンプレート](.github/workflows/codeql-analysis.yaml)はmainを囲ってるからそれに倣ってる
その辺はまあ統一したほうが良さそう。特にこだわりはない
@seito2

---

自分もこだわりないです

---

こだわりはないけど、無しで書けるんやったら無しで行きたいなーという派です

---

@tikogr @T45K 
議論をissueとして残したいのでissue立てました
こちらで議論お願いしたいです！
https://github.com/HottoCoffee/HottoCoffee/issues/14